### DISCUSSION TITLE
DDOS

### DISCUSSION THREAD

**Discussion by azouqi** [WeAreMahsaAmini/Core#481] [Ideas]
Hello friends,
I received this message from my friend please take a look and give me your ideas.
درود
کارشناس ارشد شبکه هستم.
با توجه به گستردگی فیلترینگ در کشور راه حلی ارایه می‌دهیم که همسو با این ضرب‌المثل می‌باشد.
دیگی که برای من نجوشه میخوام سر آخوند توش بجوشه.

با توجه به فعالیت‌های گروه هکری Anonymous و دیگر گروه‌های هکری، به ما ثابت شد که زیرساخت اینترنت کشور دچار مشکلات بسیار بسیار زیادی می‌باشد. در نتیجه با یک حرکت همه‌گیر از داخل کشور می‌توان تمام سایت‌های مهم کشور را از کار انداخت و بدین صورت زمینه‌ی اعتصابات سراسری را با کمترین هزینه‌ی ممکن ایجاد نمود. با توجه به اینکه در صورت حملات هکری از خارج کشور، اینترنت به صورت ملی درمی‌آید، پس باید حملات را از داخل سازماندهی کرد. روشی که در زیر توضیح داده خواهد شد، به صورت عملیاتی در ۲۰ روز گذشته تست شده و نتایج آن صد در صد رضایت‌بخش بوده است. در این روش با حملات DDOS به انواع سایت‌ها، باعث اختلال و عدم پاسخگویی از جانب سرورها شده و عملا سرویس مورد نظر از دسترس خارج می‌شود.
لازم به ذکر است که تست انجام شده در داخل کشور بوده و فقط با ۱۰ کامپیوتر این تست‌ها انجام شده است. ما توانستیم در حین تست و در کمتر از ۶۰ ثانیه ۲۵ سایت را از دسترس خارج کنیم.

ما چه کاری انجام می‌دهیم؟
ما برنامه‌ای را تولید کرده‌ایم (برنامه‌ای ویندوزی) که توسط آن و بدون نیاز به هیچ سروری، این برنامه شروع به فعالیت میکند و در زمان خاصی، مثلا ۹ صبح، تعداد محدودی درخواست (درخواست GET از صفحات سایت) به سمت یک سایت خاص ارسال میکند. چون تعداد این درخواست‌ها از طرف هر کامپیوتر محدود است، سرور تلاش می‌کند به درخواست‌ها پاسخ دهد و هیچ کدام از درخواست‌ها را رد نمی‌کند. حال توجه کنید اگر ۱۰۰.۰۰۰ نفر این نرم‌افزار را بر روی کامپیوتر خود نصب کرده باشند، به راحتی می‌توان تعداد زیادی سایت را از دسترس خارج کرد. عدد ۱۰۰.۰۰۰ نفر به نسبت معترضین خیلی خیلی کمتر می‌باشد. به طور مثال اگر این نرم‌افزار برای گوشی‌های هوشمند ایجاد گردد، عددی بالای ۱۰ میلیون نصب را به همراه خواهد داشت. این تعداد نصب یعنی ایجاد بزرگ‌ترین و وسیع‌ترین حمله‌ی DDOS در جهان که هیچ سرور و سایتی قادر به نجات از آن نخواهد بود.
به منظور مقایسه این اعداد را به خاطر داشته باشید. هم‌اکنون بزرگ‌ترین حمله DDOS دنیا در تاریخ ۲۵ ژانویه ۲۰۲۲ انجام شد و بیش از ۱۰.۰۰۰ دستگاه در آن حضور داشتند. این حمله نزدیک به ۱۵ دقیقه به طول کشید و غول‌های تکنولوژی دنیا توانستند از آن به سلامت عبور کنند.
حمله‌ی مد نظر ما با بیش از 10 میلیون اکانت انجام خواهد شد و زمان آن ۲۴ ساعت شبانه روز و ۷ روز هفته خواهد بود. بی‌شک هیچ کس و هیچ سروری توان مقابله با این حمله را نخواهد داشت.
در ضمن، در حین این حملات، مسیرهای بازشده‌ی فراوانی برای نفوذ هکرهای حرفه‌ای ایجاد خواهد شد.
هدف اصلی بانک ها و شبکه شتاب خواهد بود. در فاز دوم نیز سایت های دولتی و حکومتی به آن اضافه خواهند شد. از کار انداختن شبکه بانکی، یعنی از کار افتادن تمامی کسب و کارها و ایجاد اعتصاب سراسری.

خطرات این سیستم برای افراد چه هست؟
با توجه به احتمال ردگیری IP ها، می‌توان به هویت اشخاصی که این اپلیکیشن را بر روی گوشی خود نصب کرده‌اند، دسترسی پیدا کرد. وجود تعداد نصب بالا خطرات را برای افراد کاهش می‌دهد. حکومت نمی‌تواند ۱۰ میلیون نفر را بازداشت کند. البته اگر این اپلیکیشن به صورت سرویس بر روی گوشی نصب باشد، اگر شخص دستگیر شود، می‌تواند از وجود این اپلیکیشن ابراز بی‌اطلاعی کند. مثل اینکه این نرم‌افزار به صورت مخفی و شکل یک ویروس بر روی گوشی نصب شده است.

چه چیزی نیاز هست؟
１- اپلیکیشن قابل نصب بر روی گوشی‌های هوشمند. اگر این اپلیکیشن به صورت سرویس بر روی گوشی نصب باشد، خطرات ناشی از شناسایی IP را به شدت کاهش می‌دهد. این اپلیکیشن نیاز به هیچ دسترسی (دوربین، میکروفن و ...) نخواهد داشت و فقط به اینترنت (حتی اینترنت ملی کفایت می‌کند) نیاز خواهد داشت.
２- بهتر است که این اپلیکیشن به صورت اوپن سورس ارایه شود تا کارشناس‌های مرتبط نسبت به امن بودن آن ابراز نظر کنند.
３- این اپلیکیشن باید به صورت سراسری و توسط تمامی گروه‌ها و افراد تاثیرگزار معرفی و توسط مردم نصب شود. این سیستم بعد از معرفی نیاز است تا مدتی بر روی گوشی بوده و فعالیتی نداشته باشد تا تعداد نصب آن افزایش یابد و در زمان مناسب شروع به فعالیت کند.

سخن پایانی
بی‌شک ما با هوش و خرد جمعی خود، به همراه اتحاد و همدلی، سیاهی و ظلم را نابود خواهیم کرد.

به امید ایران پیشرفته، ثروتمند و آزاد
زن، زندگی، آزادی

**Comment by MahsaProxy** [comment]
افرادی که این اپلیکشن رو نصب می‌کنند به راحتی شناسایی میشن و این باعث میشه رژیم یه دیتابیس خوب از افرادی که بر خلافش هستند درسته کنه!
به نظرم توی این حالت سودش کمتر از ضررشه.

**Comment by azouqi** [comment]
(نظر یک کارشناس از ایران)

به نظر ایده‌ی بدی نیست. 

از نظر امنیت فردی:
با توجه به نکات زیر شاید تهدید جدی برای کسی نباشه:
۱- تعداد بالای کاربر - به نظر من راحت میشه ۱۰ میلیون یوزر براش گرفت. 
۲- این جوونایی که جلوی تیر وایسادن، احتمالا از این داستان نخواهند ترسید.
۳- میشه اپلیکیشن رو جوری پیاده کرد که شبیه یک تروجان روی گوشی باشه. در نتیجه اگر IP هم شناسایی بشه، تهدیدی جدی برای اون شخص نیست. میتونه بگه من نصی نکردم.

از نظر فنی:
به نظر من کار میکنه. چون تعداد کاربر بالا هست، میشه تعداد ریکویست رو در ثانیه آورد پایین. قطعا فایروال از کار می‌افته و DDOS اتک روی هر IP تشخیص نمیده.

از نظر کاربرد در شرایط کنونی:
قطعا میشه بانک ها و شبکه شتاب رو هدف گرفت. با مختل شدن شبکه شتاب تمام زیر ساخت مالی ساسپند میشه.
